src/me/fru1t/subedit/utils/Dialogue.kt

```diff
@@ -0,0 +1,24 @@
+package me.fru1t.subedit.utils
+
+//data class Dialogue(val fields: List<String>) {
+//    init {
+//        require(fields.size == 10) { "The length of the fields has to be 10" }
+//    }
+//
+//    fun toFormattedString(): String {
+//        return "Dialogue: ${fields.joinToString(separator = ",")}"
+//    }
+//}
+
+data class Dialogue(val layer: String, val start: String, val end: String, val style: String, val name: String,
+                    val marginL: String, val marginR: String, val marginV: String, val effect: String, val text: String) {
```

Comment:
Should `layer` and the margins be `Int` instead?

Reply:
Ah yeah... I thought like, if I do that, I should also make a new time type for `start` and `end`

Reply:
So instead I left all as strings... for consistency. What do you think?

Reply:
I'm not too worried about being consistent in terms of using the correct type for all fields. The rule of thumb (at a super super high level) is, "implement what you know you'll be using" and in this case, that's just being able to manipulate the `layer` field :)